src/main/flight/alt_hold.c

```diff
@@ -240,6 +239,9 @@ void altHoldUpdateTargetAltitude(void)
 
 void altHoldUpdate(void)
 {
+    // get current altitude and derivative
+    getCurrentAltitudeValues();
+
```

Comment:
Not needed at all, if you just get rid of the temporary copies of altitude and altitudeDerivative in the altHold struct and use `getAltitude()` and `getSpeedVertical()` directly.
```suggestion
```

Reply:
ok, done throughout, thanks